torch/ao/quantization/backend_config/_common_operator_config_utils.py

```diff
@@ -350,16 +350,35 @@ def _get_default_op_configs(dtype_configs: List[DTypeConfig]) -> List[BackendPat
         torch.nn.PReLU,
         torch.nn.functional.elu,
         torch.nn.functional.hardswish,
-        torch.nn.functional.instance_norm,
         torch.nn.functional.leaky_relu,
         torch.nn.functional.dropout,
-        torch.nn.functional.layer_norm
     ]
     for op in default_ops:
         configs.append(
             BackendPatternConfig(op)
                 .set_observation_type(ObservationType.OUTPUT_USE_DIFFERENT_OBSERVER_AS_INPUT)  # noqa: E131
                 .set_dtype_configs(dtype_configs))
+
+    configs.append(
+        BackendPatternConfig(torch.nn.functional.layer_norm)
+        .set_observation_type(ObservationType.OUTPUT_USE_DIFFERENT_OBSERVER_AS_INPUT)  # noqa: E131
+        .set_dtype_configs(dtype_configs)
+        ._set_input_type_to_index({"weight": 2, "bias": 3})
+    )
+
+    configs.append(
+        BackendPatternConfig(torch.nn.functional.group_norm)
```

Comment:
oh, I think we added quantization support for group_norm here, group_norm was not supported in the BackendConfig previously (since we removed L353 and L356 only, there were no entry for group_norm there)

Reply:
Quantized group_norm, instance_norm{n}d, layer_norm is supported in fbgemm but not qnnpack/xnnpack.  It would be great to move these config entries to the fbgemm section.

Reply:
@vkuzo Should I pull out the `nn.Modules` too for instanceNorm/LayerNorm or just the functionals?

Reply:
Sorry, I made an outdated and incorrect statement.  What I should have said is that group_norm, instance_norm{n}d, layer_norm have an optimized quantized implementation on fbgemm, and have a naive implementation on xnnpack/qnnpack.  Please ignore my comment above.